src/scss/settings/_whoWeHelp.scss

```diff
@@ -0,0 +1,137 @@
+section.whoWeHelp {
```

Comment:
That's for all CSS files: if do not need it to apply only to that element don't use that.
Also nesting should be used whenever it have value, not by default